node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java

```diff
@@ -328,6 +355,21 @@ public void setMinServiceCount(
     }
   }
 
+  @CommandMethod("tasks task <name> set hostAddress <hostAddress>")
+  public void setHostAddress(
+    @NonNull CommandSource source,
+    @NonNull @Argument("name") Collection<ServiceTask> serviceTasks,
+    @NonNull @Argument(value = "hostAddress", parserName = "ipAliasHostAddress") String hostAddress
```

Comment:
But with this there is no way to reset the host address? (aside from editing the file)